php-checks/src/main/java/org/sonar/php/checks/FunctionNameCheck.java

```diff
@@ -69,26 +71,30 @@ public void visitFunctionDeclaration(FunctionDeclarationTree tree) {
 
   private void check(NameIdentifierTree name) {
     String functionName = name.text();
-    if (!computePattern().matcher(functionName).matches() && !MAGIC_METHODS.contains(functionName)) {
-      context().newIssue(this, name, String.format(MESSAGE, functionName, computeFormat()));
+    if (!getPattern().matcher(functionName).matches() && !MAGIC_METHODS.contains(functionName)) {
+      context().newIssue(this, name, String.format(MESSAGE, functionName, getFormat()));
     }
   }
 
-  private Pattern computePattern() {
-    if (isDrupal()) {
-      return patternDrupal;
+  private Pattern getPattern() {
+    if (checkFormat(SymbolTable.Framework.DRUPAL)) {
+      return PATTERN_DRUPAL;
+    } else if (checkFormat(SymbolTable.Framework.WORDPRESS)) {
+      return PATTERN_WORDPRESS;
     }
     return pattern;
   }
 
-  private String computeFormat() {
-    if (isDrupal()) {
+  private String getFormat() {
+    if (checkFormat(SymbolTable.Framework.DRUPAL)) {
       return DEFAULT_DRUPAL;
+    } else if (checkFormat(SymbolTable.Framework.WORDPRESS)) {
+      return DEFAULT_WORDPRESS;
     }
     return format;
   }
 
-  private boolean isDrupal() {
-    return context().getFramework() == SymbolTable.Framework.DRUPAL && format.equals(DEFAULT);
+  private boolean checkFormat(SymbolTable.Framework targetFramework) {
```

Comment:
It is unclear how the name `checkFormat` relates to what the method is doing. What it is doing is:
- Check if the given framework is used
- Check if the default format was not overridden by the user (see the last sentence in https://sonarsource.atlassian.net/browse/SONARPHP-1510 to understand why this was originally introduced for Drupal)

Maybe we can separate these two:
- have a method like `isDrupalOrWordPress` as suggested above
- have a method like `isDefaultFormatOverridden` 
and then you can simply do something like `isDrupalOrWordPress() && !isDefaultFormatOverridden(). I think that would be clearer to read.